config.json

```diff
@@ -244,7 +244,38 @@
       "name": "Basics"
     }
   ],
-  "key_features": [],
+  "key_features": [
+    {
+      "icon": "widely-used",
+      "title": "Widely Used",
+      "content": "x86-64 has been the dominant ISA for personal computers since 2010."
```

Comment:
Maybe it's me, but I didn't know what an ISA is. I expect more people to also not know this acronym. Is there perhaps someway to rephrase?